Automattic/wp-calypso
# Gutenberg: refactor copied edit-post folder

#### Changes proposed in this Pull Request

* Remove copied `edit-posts` components that are now exported as part of the package.
* Removes `assets` folder and replaces it with built styles import.

#### Testing instructions

1. Navigate to http://calypso.localhost:3000/gutenberg/
2. Verify that the editor shell still works as expected and that there are no new console errors.
3. Verify that all usages of components that `edit-post` package exports are now replaced.

Fixes https://github.com/Automattic/wp-calypso/issues/26642 (to the extent that `edit-post` package now allows)
Helps with https://github.com/Automattic/wp-calypso/pull/28069
